litellm/integrations/helicone.py

```diff
@@ -31,10 +31,33 @@ def claude_mapping(self, model, messages, response_obj):
         prompt += f"{AI_PROMPT}"
         claude_provider_request = {"model": model, "prompt": prompt}
 
+        choice = response_obj["choices"][0]
```

Comment:
@maamalama does helicone have a generic way of passing messages? 

Would be easier than trying to provide openai -> provider translations for each loggging request (i assume this would also apply to other providers)

Reply:
@krrishdholakia Unfortunately, we don't have a generic gateway :( I Just made a quick fix to correctly pass anthropic request-reposnse to dedicated gateway.